source/arriving/welcome.rst

```diff
@@ -5,86 +5,35 @@ Onboarding new students
 
```

Comment:
Should we add a small welcome paragraph here?

Reply:
There is a welcome message in `index.rst` which is the first page loaded. We might want to rewrite it but can be a later TODO